grisera_api/experiment/experiment_service_ontology.py

```diff
@@ -12,9 +15,39 @@ class ExperimentServiceOntology(ExperimentService):
     """
     ontology_api_service = OntologyApiService()
 
-    def get_experiment(self, experiment_id):
-        pass
+    def get_experiment(self, experiment_label: Union[int, str], depth: int = 0):
+        """
+        Send request to ontology api to get given experiment
+        Args:
+            experiment_label (int | str): label of experiment
+            depth (int) : only for compatibility with graph_api, always set to 0
+        Returns:
+            Result of request as experiment object
+        """
+        model_id = 1
+
+        instance_response_experiment = self.ontology_api_service.get_instance(model_id=model_id,
+                                                                              class_name="Experiment",
+                                                                              instance_label=experiment_label)
+        if instance_response_experiment["errors"] is not None:
+            experiment_result = {'experiment_name': experiment_label, 'errors': instance_response_experiment["errors"]}
+            return ExperimentOut(**experiment_result)
 
+        roles_response_experiment = self.ontology_api_service.get_roles(model_id, experiment_label)
+        if roles_response_experiment["errors"] is not None:
+            return ExperimentOut(ExperimentIn(experiment_label),
+                                 errors=roles_response_experiment["errors"])
+
+        a = None
+        for prop in roles_response_experiment['roles']:
+            if prop['role'] == 'hasScenario' and prop['instance_name'] == experiment_label:
+                activity_execution_dictionary = {'id': prop['value']}
+                a = ActivityExecutionOut(**activity_execution_dictionary)
+
```

Comment:
Can `roles_response_experiment['roles']` has more than one item that meets the condition? If it can, should it only set values based on the last one? Otherwise, shouldn't we add `break`?

Reply:
There is only one item that meets condition. Added break in if statement's body.